panel/src/components/Collection/Item.vue

```diff
@@ -115,13 +120,25 @@ export default {
 		/**
 		 * If `true`, the item will be selectable via a checkbox
 		 * @since 5.0.0
+		 * @values "single", "multiple", false
 		 */
-		selecting: Boolean,
+		selecting: {
+			type: [Boolean, String],
+			default: false
+		},
```

Comment:
`Explanation:` I wanted to avoid to add another prop, but if mixing this feels too weird, we could add a `selectingMultiple: true` prop or so.